bin/bash_functions

```diff
@@ -129,7 +129,7 @@ recreate_postgresql() {
     local db_name="$1"
     local db_user="${2:-$db_name}"
     # Check if candlepin db exists and drop it if so:
-    if [[ `psql -U $db_user -tAc "SELECT 1 FROM pg_database WHERE datname='$db_name'"` == "1" ]]; then
+    if [[ `psql -U postgres -tAc "SELECT 1 FROM pg_database WHERE datname='$db_name'"` == "1" ]]; then
```

Comment:
This line may end up being a problem. The other option we have is using the specified user again, but specifying the postgres (or some other system db) so psql doesn't complain about the role's DB not existing; such as when using -U candlepin and checking for "testdbname".